presto-main/src/main/java/com/facebook/presto/security/ViewAccessControl.java

```diff
@@ -34,7 +34,7 @@ public ViewAccessControl(AccessControl delegate, QualifiedObjectName viewName)
     @Override
     public void checkCanSelectFromTable(TransactionId transactionId, Identity identity, QualifiedObjectName tableName)
     {
-        delegate.checkCanCreateViewWithSelectFromTable(transactionId, identity, viewName, tableName);
+        delegate.checkCanCreateViewWithSelectFromTable(transactionId, identity, tableName);
```

Comment:
I believe you can now remove `viewName` from this class since this was the only usage